report/sections/system-design.tex

```diff
@@ -34,6 +35,39 @@ \subsection{Model}
   \caption{High-level state machine of the system.}
   \label{fig:state_machine}
 \end{figure*}
+\subsubsection{Mobile platform}
+The model used in this case is the unicycle model, due to the difrential steering. This is because of the mobile platform has only two wheels/trucks and it is not able to apply any steering angel to it's wheels. The only way this robot can change orientation is by giving different velocity on each wheel-driving servo on left- and right- side. With this feature it is also possible to change the orientation of the mobile platform without changing the position of the platform. I.e. the robot is able to spin while the right-hand sidewheels have the same velocity as the left-hand side wheels, in opposite direction. 
```

Comment:
Remainder of the report is written with one line per sentece. Refactor to that format. Among other reasons, this makes it easier to review commits.

Reply:
Also:
* difrential -> differential
* angel -> angle
* it's wheels -> its wheels

Reply:
> Remainder of the report is written with one line per sentece. Refactor to that format. Among other reasons, this makes it easier to review commits.

What do you mean with one line per sentence? should i insert a new line after every sentence?